pibooth/booth.py

```diff
@@ -375,6 +376,10 @@ def main_loop(self):
 def main():
     """Application entry point.
     """
+    if hasattr(multiprocessing, 'set_start_method'):
+        # Avoid use 'fork': safely forking a multithreaded process is problematic
+        multiprocessing.set_start_method('spawn')
```

Comment:
I don't understand how this can have an effect, is the multiprocessing package used by pluggy?

Reply:
The multiprocessing module is used by the `pibooth/pictures/pool.py` to generate each frame of the animated picture in a different process, however the choice of the method used to fork the main process shall be protected by the `if __name__ == '__main__'` condition has explained in the documentation, that is why this is done at this step.